starter-project-backend-g32/src/models/Comment.ts

```diff
@@ -0,0 +1,16 @@
+import mongoose, {Schema, Document} from 'mongoose'
+
+
+export interface IComment extends Document {
+    article:string,
+    commenter:string,
+    body:string,
+}
+
+const CommentSchema:Schema = new Schema({
+    article:{type:String,required:true},
```

Comment:
better to use pritter(make more readable) separate type and required in a different line, or use Vscode or other extensions to perform that.